lntest/itest/lnd_single_hop_invoice_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"encoding/hex"
 	"time"
 
+	"github.com/lightningnetwork/lnd/lnwire"
```

Comment:
this should be moved down with the other imports

Reply:
Fixed